client/my-sites/site-settings/media-settings.jsx

```diff
@@ -260,7 +249,11 @@ export default connect(
 		const siteInDevMode = isJetpackSiteInDevelopmentMode( state, selectedSiteId );
 		const sitePlanSlug = getSitePlanSlug( state, selectedSiteId );
 		const moduleUnavailableInDevMode = isJetpackModuleUnavailableInDevelopmentMode( state, selectedSiteId, 'photon' );
-		const isVideoPressAvailable = includes( plansIncludingVideoPress, sitePlanSlug );
+		const isVideoPressAvailable = (
+			hasFeature( state, selectedSiteId, FEATURE_VIDEO_UPLOADS ) ||
+			hasFeature( state, selectedSiteId, FEATURE_VIDEO_UPLOADS_JETPACK_PREMIUM ) ||
+			hasFeature( state, selectedSiteId, FEATURE_VIDEO_UPLOADS_JETPACK_PRO )
+		);
```

Comment:
In any case, I'm fine with going with the other approach as well, so I'll open a PR for it.